applications/nrf5340_audio/dfu/empty_net/src/main.c

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+
+void main(void)
+{
+	;
```

Comment:
I believe the NET core should be put in sleep if not used, or is this entire file a dummy?

Reply:
This is dummy application. We don't program it.

Reply:
Add a comment in main. Suggest:
```suggestion
	; /* Dummy application. Unused main */
```